curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/615389bd81347947ea7ba896.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 615389bd81347947ea7ba896
-title: Step 11
+title: Step 15
```

Comment:
This is a duplicate of step 14.
I think you should delete this step since it is a duplicate.
You can use the [step tools](https://contribute.freecodecamp.org/#/how-to-work-on-practice-projects?id=delete-step) for that

Reply:
@naomi-lgbt, what do you think?